application/views.py

```diff
@@ -6,7 +6,7 @@
 from django.urls import reverse
 
 
-def new_application(request):
+def new_application(request, user_type):
     if request.method == "POST":
         # TODO user_id will be replaced by sessions
```

Comment:
This should be done since sessions is already implemented and is available on develop

Reply:
this is related to another story of creating and submitting applications. Will be creating a separate PR for implementing sessions.

Reply:
Can it not be incorporated into this PR itself? Since sessions changes has to be made on the same page, you can make the changes on the same PR.

Reply:
These are two different flows, although the changes are in the same files.